docs/sources/setup-grafana/installation/docker/index.md

```diff
@@ -8,261 +8,334 @@ weight: 200
+`name` = assign a logical name to the container, for example, `grafana`
 
-These images are based on [Ubuntu](https://ubuntu.com/), available in [the Ubuntu official image](https://hub.docker.com/_/ubuntu). It is an alternative image for those who prefer an [Ubuntu](https://ubuntu.com/) based image and/or are dependent on certain tooling not available for Alpine.
+`grafana/grafana-enterprise` = the image to run in the container
 
-## Run Grafana
+### Stop the Grafana container
 
-You can run the latest Grafana version, run a specific version, or run an unstable version based on the main branch of the [grafana/grafana GitHub repository](https://github.com/grafana/grafana).
+To stop the Grafana container, run the following command:
 
-### Run the latest stable version of Grafana
+```bash
+# The `docker ps` command shows the processes running in Docker
+docker ps
 
-> **Note:** If you are on a Linux system, you might need to add `sudo` before the command or add your user to the `docker` group.
+# This will display a list of containers that looks like the following:
+CONTAINER ID   IMAGE  COMMAND   CREATED  STATUS   PORTS    NAMES
+cd48d3994968   grafana/grafana-enterprise   "/run.sh"   8 seconds ago   Up 7 seconds   0.0.0.0:3000->3000/tcp   grafana
 
-```bash
-docker run -d -p 3000:3000 grafana/grafana-enterprise
+# To stop the grafana container run the command
+# docker stop CONTAINER-ID or use
+# docker stop NAME, which is `grafana` as previously defined
+docker stop grafana
 ```
 
-### Run a specific version of Grafana
+### Save your Grafana data
 
-> **Note:** If you are on a Linux system, you might need to add `sudo` before the command or add your user to the `docker` group.
+When you use Docker containers, their data is temporary by default. If you don't specify where to store the information, all the Grafana data will be lost when you stop the Docker container. To avoid losing your data, you can set up [persistent storage](https://docs.docker.com/storage/volumes/) or [bind mounts](https://docs.docker.com/storage/bind-mounts/) for your container.
 
-```bash
-docker run -d -p 3000:3000 --name grafana grafana/grafana-enterprise:<version number>
-```
+> **Note:** Though both methods are similar, there is a slight difference. If you want your storage to be fully managed by Docker and accessed only through Docker containers and the Docker CLI, you should choose to use persistent storage. However, if you need full control of the storage and want to allow other processes besides Docker to access or modify the storage layer, then bind mounts is the right choice for your environment.
+
+#### Use persistent storage (recommended)
+
+It is recommended to have persistent storage because without it, all data will be lost once the container is shut down. Use this method when you deploy in a production environment or when the data is critical and want the Docker service to fully manage the storage volume.
```

Comment:
```suggestion
It is recommended to have persistent storage because without it, all data will be lost once the container is shut down. Use this method when you want the Docker service to fully manage the storage volume.
```